diff --git a/node_modules/urql/dist/urql.es.js b/node_modules/urql/dist/urql.es.js
index c2ef1c7..6f09fe5 100644
--- a/node_modules/urql/dist/urql.es.js
+++ b/node_modules/urql/dist/urql.es.js
@@ -70,11 +70,10 @@ var hasDepsChanged = (e, r) => {
   return !1;
 };
 
-var p = r.__SECRET_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED;
 
 function deferDispatch(e, r) {
-  if ("production" !== process.env.NODE_ENV && p && p.ReactCurrentOwner && p.ReactCurrentOwner.current) {
-    Promise.resolve(r).then(e);
+  if (process.env.NODE_ENV !== 'production') {
+    Promise.resolve().then(() => e(r));
   } else {
     e(r);
   }
